.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,40 @@
+---
+name: Bug report
+about: Did You find a bug in PMDK? Please let us know.
+
+---
+<!-- 
+Before creating new issue, please ensure that the issue wasn't allready created
+  * Search: https://github.com/pmem/issues/issues
+
+Note that if you do not provide enough information to reproduce the issue, we may not be able to take action on your report.
+-->
+
+# ISSUE: <!-- fill the title of issue -->
+
+## Environment Information 
+ 
+- Package version(s): <!-- fill this out -->
+- OS versions:        <!-- fill this out -->
+
```

Comment:
Maybe we should add here also "\<!-- fill in also other useful environment data -->"

Reply:
Done.